### ISSUE TITLE
Accessibility Issue : Fast paas issue of Tag picker when max limit items are selected. aria-input-fieldname issue.

### ISSUE THREAD

**Issue by Sandhya-aluri** [microsoft/fluentui#15547]
### Environment Information

- **Package version(s)**:  "@fluentui/react": "^7.126.0",
- **Browser and OS versions**: chrome and Windows 10 

### Describe the issue:
Selecting the max limit items in Tag picker and click the fast paas on. aria-input-fieldname issue is raised.

#### Actual behavior:
Selecting the max limit items in Tag picker and click the fast paas on. aria-input-fieldname.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/48669559/96256867-09b25700-0fd7-11eb-9501-4c2a9b5cc7bd.png)
<img width="424" alt="fastpaasTagpicker" src="https://user-images.githubusercontent.com/48669559/96256927-20f14480-0fd7-11eb-90ab-ccb38337cf63.PNG">

 
<div role="combobox" aria-expanded="false" aria-haspopup="dialog" class="ms-FocusZone css-162" data-focuszone-id="FocusZone150">

#### Expected behavior:
Selecting the max limit items in Tag picker , fast paas should be with no errors.

**Comment by paulgildea** [comment]
@Sandhya-aluri Thanks for filing an issue.

@smhigley FYI
